src/components/LandDetailPage/LandDetailPage.tsx

```diff
@@ -11,6 +11,7 @@ import Profile from 'components/Profile'
 import Scene from './Scene'
 import { Props, State } from './LandDetailPage.types'
 import './LandDetailPage.css'
+import { env } from 'decentraland-commons'
```

Comment:
nip tick: cam we move this import to the top? we usually have the modules on top and the relative imports below